AsyncDisplayKitTests/ASDisplayNodeTests.m

```diff
@@ -2185,8 +2187,10 @@ - (void)testThatBackgroundLayoutSpecOrdersSubnodesCorrectly
     // The inset spec here is crucial. If the nodes themselves are children, it passed before the fix.
     return [ASBackgroundLayoutSpec backgroundLayoutSpecWithChild:overlay background:[ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsZero child:underlay]];
   };
-  [node layoutThatFits:ASSizeRangeMake(CGSizeMake(100, 100))];
-  node.frame = (CGRect){ .size = node.calculatedSize };
+  
+  ASDisplayNodeSizeToFitSize(node, CGSizeMake(100, 100));
```

Comment:
No I don't think so as the cool thing is we can do measurement on the background thread without doing layout. So if we would add `layoutIfNeeded` to `ASDisplayNodeSizeToFitSizeRange` we cannot call that anymore on a background thread as layout needs to happen on main. Look into the `testMeasurementInBackgroundThreadWithLoadedNode` test for example